src/tooltip/tooltip.js

```diff
@@ -302,6 +299,18 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
 
             attrs.$observe( prefix+'Title', function ( val ) {
               ttScope.title = val;
+              if (prefix === 'popover') {
+                $timeout(positionTooltip, 0, false);
+              }
+            });
+
+            attrs.$observe( prefix + 'Placement', function () {
+              if (ttScope.isOpen) {
+                $timeout(function () {
+                  prepPlacement();
+                  show()();
```

Comment:
Ah right, I missed that check, but there's also when the value is a blank string, see above at line #288: `if (!val && ttScope.isOpen ) {`